src/blockdevs.rs

```diff
@@ -200,5 +206,7 @@ fn test_lsblk_smoke() {
     let devs = BlockDevice::list().expect("Valid lsblk");
     for dev in devs.iter().filter(|d| d.is_part()) {
         let _ = dev.capacity().unwrap().unwrap();
+        let _ = dev.sysfs().unwrap();
+        let _ = dev.disk_name().unwrap();
```

Comment:
Panic here?

Reply:
hm? for the test case yeah? or should I change it to just throw error up

Reply:
> `.filter(|d| d.is_part())`

but in `disk_name()`, you panic when `self.is_part()` is true.

Reply:
nevermind, brain fog